src/hooks/postrun/plugin-postrun.js

```diff
@@ -1,16 +1,16 @@
 /* eslint-disable no-console */
 const chalk = require('chalk');
 
-const AUTOCOMLETE_WARNING = `If you’re using autocomplete, you’ll need to run ${chalk.bold(
+const AUTOCOMLETE_UPDATE_WARNING = `If you’re using autocomplete, you’ll need to run ${chalk.bold(
   'twilio autocomplete',
-)} after installing a plugin and then open a new terminal window. The CLI needs to re-build its cache.`;
+)} after an update and then open a new terminal window. The CLI needs to re-build its cache.`;
 const AUTOCOMLETE_ALERT = `If you are running bash or zsh on macOS or Linux, you can run one of the two commands below (as appropriate for the shell you are using): \n '${chalk.bold(
   '1) twilio autocomplete bash',
 )}' or \n '${chalk.bold('2) twilio autocomplete zsh')}' `;
 
 module.exports = async function pluginPostRun(options) {
-  if (options.Command.id === 'plugins:update') {
-    console.warn(chalk.yellowBright(` » ${AUTOCOMLETE_WARNING}`));
+  if (options.Command.id === 'plugins:update' || options.Command.id === 'update') {
```

Comment:
[Q] When `options.Command.id === 'update'` condition will be executed?

Reply:
This is the 'twilio update' command which I'm adding as part of this pull request. It updates the CLI for us. And once that is done, we need to warn the users about autocomplete as the cache needs to be rebuilt.